src/main/java/com/jcabi/s3/Bucket.java

```diff
@@ -108,7 +108,28 @@ void remove(@NotNull(message = "S3 key can't be NULL") String key)
         String pfx) throws IOException;
 
     /**
-     * Prefixed.
+     * Creates bucket with specified origin bucket and prefix.
+     *
+     * <p>Basically this class is used to cut off ocket keys of underlying
+     * bucket by some string known as prefix. If key is not started
+     * with prefix, it will be omitted
+     *
+     * <p>Example of usage:
+     * <pre>
+     *     final Region region = new MkRegion(
+     *         new TemporaryFolder().newFolder()
+     *     );
+     *     final Bucket bucket = region.bucket("test");
+     *     new Ocket.Text(bucket.ocket("a/first.txt")).write("");
+     *     new Ocket.Text(bucket.ocket("a/b/hello.txt")).write("");
+     *     new Ocket.Text(bucket.ocket("a/b/f/2.txt")).write("");
+     *     Bucket.Prefixed prefixed = new Bucket.Prefixed(
+     *         bucket, "a/b/"
+     *     );
+     *     Iterable<String> list = prefixed.list(
+     *         ""
+     *     ); // contains "hello.txt" and "f/2.txt"
```

Comment:
what's the point of this indentation? see how javadocs are formatted, for example here: https://github.com/jcabi/jcabi-s3/blob/master/src/main/java/com/jcabi/s3/Ocket.java#L49-L62